scripts/setup-secrets.sh

```
@@ -0,0 +1,126 @@
+    echo ""
+    
+    if [ -z "$secret_value" ]; then
+        echo "⚠️  Empty value, skipping"
+        echo ""
+        return
+    fi
+    
+    # Add secret using gh CLI
+    echo "$secret_value" | gh secret set "$secret_name" --repo "$REPO"
+    
+    if [ $? -eq 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)